src/request/containers/RequestDetailPanelDataContainer.ts

```diff
@@ -0,0 +1,21 @@
+import { DataComponent, IDataComponentProps } from '../components/RequestDetailPanelData';
+import { IRequestState } from '../stores/IRequestState';
+import { getTotalOperationCount } from '../selectors/RequestDetailDataSelectors';
+
+import { connect } from 'react-redux';
+
+function mapStateToProps(state: IRequestState): IDataComponentProps {
+    return {
+        totalOperationCount: getTotalOperationCount(state)
+    };
+}
+
+function mapDispatchToProps(dispatch) {
+    return {
+    };
+}
+
+export = connect(
+    mapStateToProps,
+    mapDispatchToProps
+)(DataComponent);
```

Comment:
Not sure; the documentation doesn't specify.  It looks like I could simply pass `{}` however (which `react-redux` treats as an object of action creators and where it would wrap each property in a dispatch-passed function and merge with the component's properties, but since there are none, essentially nothing would happen).  It's less code, which would is generally good, but it's also less explicit (that there are no dispatch properties), which can sometimes hinder understanding.